tornado-api/src/main/java/uk/ac/manchester/tornado/api/common/PrebuiltTaskPackage.java

```diff
@@ -70,4 +71,12 @@ public boolean isPrebuiltTask() {
     public int[] getAtomics() {
         return atomics;
     }
+
+    public void withClass(Class<?> klass) {
+        this.klass = klass;
+    }
+
+    public Class<?> getClassJar() {
```

Comment:
and this `getKlass`?

Reply:
In Java, there is a method called `getClass`, so to avoid miss-understanding about the functionality of each, I named it `ClassJar`